src/main/java/com/sfuentes/Employee.java

```diff
@@ -0,0 +1,37 @@
+package com.sfuentes;
+
+public class Employee {
+
+  int employeeId;
+  final double baseSalary = 10000.00;
```

Comment:
Optional.
you can add underscores to separate the zeroes for better readability:
final double baseSalary = 10_000.00;

Reply:
fixed